yw_website/yw_website/apps/website/templates/pages/edit_page.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'extendable/__base_template.html' %}
+{% load static %}
+
+{% block main_content %}
+<form id="EditForm" method="POST">
+	{% csrf_token %}
+	<div class="edit" style="margin-left: 3%">
```

Comment:
Should keep the CSS in a different file?

Reply:
> Should keep the CSS in a different file?
yes